sql-cloudant/src/main/scala/org/apache/bahir/cloudant/common/JsonStoreDataAccess.scala

```diff
@@ -125,6 +125,19 @@ class JsonStoreDataAccess (config: CloudantConfig)  {
     val clRequest: HttpRequest = getClRequest(url, postData)
 
     val clResponse: HttpResponse[String] = clRequest.execute()
+          
+    //BAHIR-130 start
+    //In cloudant Lite number of requests pre second are limited.
+    //Just wait and try again
+    clResponse.code match {
+        case 429 => {
+            Thread sleep 1000
```

Comment:
For each retry we should consider increasing the sleep 'backoff' time.  Use something like `long sleepTime = initialSleep * Math.round(Math.pow(2, attempt))` to double the sleep time based on the number of retry attempts.  Our other libraries start with an initial sleep time of 250 ms.  Here's the [retry method](https://github.com/cloudant/java-cloudant/blob/515cf156751cd67d714a206d1bc39a5489f8c156/cloudant-http/src/main/java/com/cloudant/http/interceptors/Replay429Interceptor.java#L83.) in our java library that may be helpful.